include/enoki/texture.h

```diff
@@ -10,29 +10,29 @@
     license that can be found in the LICENSE file.
 */
 
+#include <array>
+#include <enoki-jit/texture.h>
+#include <enoki/dynamic.h>
+#include <enoki/idiv.h>
 #include <enoki/jit.h>
 #include <enoki/tensor.h>
-#include <enoki/dynamic.h>
-#include <enoki-jit/texture.h>
-#include <array>
 
 #pragma once
 
 NAMESPACE_BEGIN(enoki)
 
-enum class FilterMode : uint32_t {
-    Nearest = 0,
-    Linear = 1
-};
+enum class FilterMode : uint32_t { Nearest = 0, Linear = 1 };
+
+enum class WrapMode : uint32_t { Repeat = 0, Clamp = 1, Mirror = 2 };
```

Comment:
Let's extend those on multiple lines
```suggestion
enum class FilterMode : uint32_t { 
    Nearest = 0, 
    Linear = 1 
};

enum class WrapMode : uint32_t { 
    Repeat = 0, 
    Clamp = 1, 
    Mirror = 2 
};
```

Reply:
Could also add short comments / descriptions